suitcase/spec.py

```diff
@@ -1013,7 +1015,7 @@ def _get_plan_name(start):
             "Until this feature is implemented, we will be using the sequence "
             "number as the motor position".format(plan_name))
         return _NOT_IMPLEMENTED_SCAN
-    return _SPEC_SCAN_NAMES[_BLUESKY_PLAN_NAMES.index(plan_name)]
+    return _BLUESKY_PLAN_NAMES[plan_name]
```

Comment:
I thought we talked about using `get(plan_name, plan_name)` to let plan names pass without renaming instead of using 'Other'.